code/modules/emoji/emoji_parse.dm

```diff
@@ -19,7 +19,7 @@ var/list/emojis
 			if(search)
 				emoji = lowertext(copytext(text, pos+1, search))
 				if(emoji in emojis)
-					parsed += " <img class=icon src=\ref['icons/emoji.dmi'] iconstate='[emoji]'>"
+					parsed += bicon(icon('icons/emoji.dmi', emoji))
```

Comment:
It appears to take all the frames and throw them into a grid
![](https://cloud.githubusercontent.com/assets/5211576/23095295/60256cb4-f5d5-11e6-8bc1-75fe42acdaa2.png)

Best solution is to ban animated emoji and remove the extra frames from the emoji.dmi, which are supermatter and forcesword